### DISCUSSION TITLE
WWW does not work for my instance

### DISCUSSION THREAD

**Discussion by n33d4n4m3** [mastodon/mastodon#21518] [Q&A]
Hello folks,

I unfortunately encountered a problem while setting up my Mastodon instance.
My instance is fully set up, only the SSL certificate is not working properly.
The instance runs under _example.com_, however when I try to reach my instance via _**www**.example.com_ my brwoser displays **NET::ERR_CERT_COMMON_NAME_INVALID** error. I've already tried using the NGINX configuration files to redirect from the WWW domain to the NON-WWW domain, but that doesn't work.

Anyone have an idea?
Thanks in advance!

**Comment by pumpkinlawrence** [comment]
Did you ever get this problem figured out? I have the same problem. Thank you.
